nerdlandbot/helpers/TranslationHelper.py

```diff
@@ -12,3 +13,25 @@ async def get_culture_from_context(ctx: commands.Context) -> str:
     guild_id = ctx.guild.id
     guild = await get_guild_data(guild_id)
     return guild.culture
+
+def foemp_or_schat():
+    # if it's weekend (5 or 6) the bot will say schatje instead of foemp.
+    adjective = "foemp"
+    if datetime.now().weekday() == 0 or datetime.now().weekday() == 1:
```

Comment:
Think this is still set to 0 or 1 for testing?

Reply:
Changed the numbers to constants